src/Mod/Sketcher/Gui/Utils.h

```diff
@@ -119,6 +119,9 @@ inline bool isEdge(int GeoId, Sketcher::PointPos PosId)
 // Return counter-clockwise angle from horizontal out of p1 to p2 in radians.
 double GetPointAngle (const Base::Vector2d &p1, const Base::Vector2d &p2);
 
+// Set the two points on circles at minimal distance
+bool GetCirclesMinimalDistance(const Part::GeomCircle *circle1, const Part::GeomCircle *circle2, Base::Vector3d *point1, Base::Vector3d *point2);

```

Comment:
The return value is never used.

Reply:
Could be used elsewhere, maybe in the coming line to circle distance or for the arc generalization. It has no charge to keep it.

Reply:
It doesn't harm the code. But it can bring confusion to one reading the code. Very minor though. :)

Reply:
This function could difficultly be reused for an arc o a point as type is hardcoded, and there is no inheritance relationship. At best it might be templated.

I think YAGNI, but if you insist, I would at least update the documentation (use three slashes instead of two and indicate what it returns), then in most IDEs the developer using the code will be notified by the IDE.